certbot/main.py

```diff
@@ -215,6 +215,10 @@ def _find_cert(config, domains, certname):
               RenewableCert instance or None.
     """
     action, lineage = _find_lineage_for_domains_and_certname(config, domains, certname)
+    if action == "renew":
```

Comment:
> we can remove the machinery that asks "looks like you have a cert with these domains already, want to renew instead?"

What if a user of `certbot-auto` is relying on this behavior for a command in their crontab?